doc/versions.json

```diff
@@ -30,12 +31,12 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "2.0.99",
-      "nrfxlib": "2.0.99",
+      "ncs": "2.1.0",
+      "nrfxlib": "2.1.0",
       "zephyr": "3.1.99",
       "mcuboot": "1.9.99",
-      "nrfx": "2.8",
-      "tfm": "1.5.0",
+      "nrfx": "2.9",
+      "tfm": "1.6.0",
       "matter": ""
     },
     "2.0.2": {
```

Comment:
We should also have a set like this for 2.1.0.

Reply:
Updated.